spec/dish_spec.rb

```diff
@@ -0,0 +1,17 @@
+require './lib/dish'
+require 'money'
+
+describe Dish do
+  let(:my_dish) { Dish.new("Rice", 100) }
+  it 'can be initialized with a name' do
+    expect(my_dish.name).to eq("Rice")
```

Comment:
I like that you have chosen to test the data structure and the presentation of the data separately in this class